session/manager_http.go

```diff
@@ -130,6 +130,10 @@ func (s *ManagerHTTP) IssueCookie(ctx context.Context, w http.ResponseWriter, r
 	cookie.Values["session_token"] = session.Token
 	cookie.Values["expires_at"] = session.ExpiresAt.UTC().Format(time.RFC3339Nano)
 
+	for _, mod := range cookieModifiers {
+		mod(cookie)
+	}
```

Comment:
Remove this and just set the nonce at all times. We want immutable cookies _always_, not just sometimes. And it makes testing easier :)

Reply:
Fixed